### PR TITLE
feat(scripts): Add script to create a `@discordjs` package

### PR DESCRIPTION
**Pull request by suneettipirneni**
**Please describe the changes this PR makes and why it should be merged:**

Streamlines the process of creating new packages under the `@discordjs` org.

To use simply run `yarn createPackage <package-name> [package-desription]` in the root directory.

### CODE REVIEW

File: packages/scripts/src/template/template.package.json [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+		"downlevel-dts": "^0.10.1",
+		"eslint": "^8.24.0",
+		"eslint-config-neon": "^0.1.33",
+		"mock-socket": "^9.1.5",
+		"prettier": "^2.7.1",
+		"tsup": "^6.2.3",
+		"typescript": "^4.8.3",
+		"undici": "^5.10.0",
+		"vitest": "^0.23.4",
+		"zlib-sync": "^0.1.7"
```
Comment (imranbarbhuiya) [inline packages/scripts/src/template/template.package.json:71]:
```suggestion
		"@discordjs/docgen": "workspace:^",
		"@favware/cliff-jumper": "^1.8.7",
		"@microsoft/api-extractor": "^7.31.2",
		"@types/node": "^16.11.60",
		"@vitest/coverage-c8": "^0.23.4",
		"cross-env": "^7.0.3",
		"downlevel-dts": "^0.10.1",
		"eslint": "^8.24.0",
		"eslint-config-neon": "^0.1.33",
		"prettier": "^2.7.1",
		"tsup": "^6.2.3",
		"typescript": "^4.8.3",
		"undici": "^5.10.0",
		"vitest": "^0.23.4"
```

File: packages/scripts/package.json [hunk 2]
```diff
@@ -44,12 +44,16 @@
 	"homepage": "https://discord.js.org",
 	"dependencies": {
 		"@discordjs/api-extractor-utils": "workspace:^",
+		"@iarna/toml": "^2.2.5",
 		"@microsoft/api-extractor-model": "7.24.0",
 		"@microsoft/tsdoc": "0.14.1",
 		"@microsoft/tsdoc-config": "0.16.1",
 		"commander": "^9.4.1",
+		"fs-extra": "^10.1.0",
+		"toml": "^3.0.0",
```
Comment (iCrawl) [inline packages/scripts/package.json:53]: 2 toml packages?

File: packages/scripts/src/template/template.package.json [hunk 3]
```diff
@@ -0,0 +1,65 @@
+{
+	"name": "@",
+	"version": "0.1.0",
+	"description": "",
+	"scripts": {
+		"test": "vitest run",
+		"build": "tsup",
+		"lint": "prettier --check . && cross-env TIMING=1 eslint src __tests__ --ext mjs,js,ts",
+		"format": "prettier --write . && cross-env TIMING=1 eslint src __tests__ --ext mjs,js,ts --fix",
```
Comment (iCrawl) [inline packages/scripts/src/template/template.package.json:9]: Have to re-add the `cross-env` package as a dev-dep for this to work.

**Review by iCrawl — CHANGES_REQUESTED** [review]
Missed a few things

**Review by SpaceEEC — APPROVED** [review]
This formats `.github/labeler.yml` and `.github/labels.yml` (removes unneeded quotes), which is probably fine.

So LGTM

File: packages/scripts/src/template/template.package.json [hunk 4]
```diff
@@ -0,0 +1,66 @@
+{
+	"name": "{name}",
+	"version": "0.1.0",
+	"description": "",
```
Comment (iCrawl) [inline packages/scripts/src/template/template.package.json:4]: This is not actually filling in any values 👀

File: package.json [hunk 5]
```diff
@@ -11,7 +11,8 @@
 		"fmt": "turbo run format --parallel",
 		"postinstall": "is-ci || husky install",
 		"docs": "turbo run docs --parallel",
-		"update": "yarn upgrade-interactive"
+		"update": "yarn upgrade-interactive",
+		"createPackage": "node packages/scripts/src/packageScript.mjs"
```
Comment (iCrawl) [inline package.json:15]:
```suggestion
		"create-package": "node packages/scripts/src/packageScript.mjs"
```

File: packages/scripts/src/template/template.package.json [hunk 6]
```diff
@@ -0,0 +1,66 @@
+{
+	"name": "@discordjs/{name}",
+	"version": "0.1.0",
+	"description": "",
+	"scripts": {
+		"test": "vitest run",
+		"build": "tsup",
+		"lint": "prettier --check . && cross-env TIMING=1 eslint src __tests__ --ext mjs,js,ts",
+		"format": "prettier --write . && cross-env TIMING=1 eslint src __tests__ --ext mjs,js,ts --fix",
+		"docs": "downlevel-dts dist docs/dist --to=3.7 && docgen -i src/index.ts -c docs/index.json -o docs/docs.json --typescript && api-extractor run --local",
```
Comment (iCrawl) [inline packages/scripts/src/template/template.package.json:10]:
```suggestion
		"docs": "downlevel-dts dist docs/dist --to=3.7 && api-extractor run --local",
```

File: packages/scripts/src/template/template.package.json [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+		"type": "git",
+		"url": "git+https://github.com/discordjs/discord.js.git"
+	},
+	"bugs": {
+		"url": "https://github.com/discordjs/discord.js/issues"
+	},
+	"homepage": "https://discord.js.org",
+	"dependencies": {},
+	"devDependencies": {
+		"@discordjs/docgen": "workspace:^",
```
Comment (iCrawl) [inline packages/scripts/src/template/template.package.json:46]:
```suggestion
```

File: packages/scripts/src/template/template.package.json [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+		"@types/node": "^16.11.60",
+		"@vitest/coverage-c8": "^0.23.4",
+		"downlevel-dts": "^0.10.1",
+		"eslint": "^8.24.0",
+		"eslint-config-neon": "^0.1.33",
+		"prettier": "^2.7.1",
+		"tsup": "^6.2.3",
+		"typescript": "^4.8.3",
+		"vitest": "^0.23.4",
+		"cross-env": "^7.0.3"
```
Comment (iCrawl) [inline packages/scripts/src/template/template.package.json:58]: This should be sorted correctly.

**Review by iCrawl — CHANGES_REQUESTED** [review]
Last batch before merge.

**Review by iCrawl — APPROVED** [review]

**Pull request merged**